choreography/src/delivery/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM microsoft/dotnet:2.2-aspnetcore-runtime as base
+FROM mcr.microsoft.com/dotnet/core/aspnet:2.2 as base
```

Comment:
By the way, I think it is just safe to remove the `Dockerfile`. Not seeing this being relevant for this Cloud Design Pattern.

Reply:
Ah, I'm just observing couple of spots where `containers` are mentioned as part of the story telling. Please let me mark this as resolved.